Commons/src/main/java/eu/neclab/ngsildbroker/commons/tools/HttpUtils.java

```diff
@@ -890,7 +939,7 @@ public static String getTriggerReason(int triggerReason) {
 
 	public static Uni<RestResponse<Object>> generateQueryResult(HttpServerRequest request, QueryResult queryResult,
 																String options, String geometryProperty, int acceptHeader, boolean count, int limit, LanguageQueryTerm lang,
-																Context context, JsonLDService ldService) {
+																Context context, JsonLDService ldService,List<String> omitList,String pick) {
```

Comment:
Pick parameter should be List of String.

Reply:
done